core/tests/test_utils.py

```diff
@@ -263,6 +370,22 @@ def _send_post_request(
     def post_email(
             self, recipient_email, sender_email, subject, body, html_body=None,
             expect_errors=False, expected_status_int=200):
+        """EmailMessage Class is used for creating the email message.
```

Comment:
Don't talk about implementation details; say what this function does. In this case, it should be sufficient to say:

"""Sends an email with the given parameters."""

Reply:
done

Reply:
done